src/main/java/appeng/core/features/registries/P2PTunnelRegistry.java

```diff
@@ -73,7 +73,6 @@ public void configure() {
         final IBlocks blocks = definitions.blocks();
         final IParts parts = definitions.parts();
 
-        this.addNewAttunement(blocks.iface(), TunnelType.ITEM);
         this.addNewAttunement(parts.iface(), TunnelType.ITEM);
```

Comment:
The interface part should also attune to the new tunnel

Reply:
Done.